scripts/deploy.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Script to deploy skynet-js to an hns domain.
+ *
+ * Example usage: SKYNET_JS_DEPLOY_SEED="..." node ./scripts/deploy.js
+ *
+ * You can generate the required seed with `genKeyPairAndSeed`.
+ *
+ * The first time you run this for a given hns domain, there won't be any data
+ * on the domain. Setting `skipDownload` will skip the download. After the
+ * upload, set the TXT record for the hns domain to the resulting resolver
+ * skylink.
+ */
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const {
+  genKeyPairFromSeed,
+  SkynetClient,
+  stringToUint8ArrayUtf8,
+  uriSkynetPrefix,
+} = require("@skynetlabs/skynet-nodejs");
+
+const fs = require("fs");
+const fse = require("fs-extra");
+const process = require("process");
+const tar = require("tar-fs");
+
+// The env var with the secret seed phrase to deploy with.
+const deploySeedEnvVar = "SKYNET_JS_DEPLOY_SEED";
+// The hns domain to deploy to.
+const hnsDomain = "skynet-js";
+// The location of the bundle to deploy. Must be a folder.
+const bundlePath = "bundle";
+// Location of package.json, used to get the latest version.
+const packageJson = "../package.json";
+// Set to true to skip the download.
+const skipDownload = false;
+// Set to true to skip the upload.
+const skipUpload = false;
+
+const dataKey = "skynet-js";
+const versionsDir = "versions";
+const versionsTarFile = `${versionsDir}.tar`;
+
+(async () => {
+  const client = new SkynetClient("https://siasky.net");
```

Comment:
nit: Since this is a script, it would be nice to be able to override some of the values here without editing the script itself, e.g. skynet portal URL, `skipDownload`. Maybe also `skipUpload`, although I'm not sure why we'd want that, except for debugging.

Reply:
Great idea, implemented. I didn't make all the values configurable from the CLI, just the useful ones. I also added a `--first-time` flag to automate that process.